overlay/frameworks/base/core/res/res/values/config.xml

```diff
@@ -565,4 +565,10 @@
     <!-- Whether device has dash charging support -->
     <bool name="config_hasDashCharger">true</bool>
 
+    <!-- Controls the navigation bar interaction mode:
+         0: 3 button mode (back, home, overview buttons)
+         1: 2 button mode (back, home buttons + swipe up for overview)
+         2: gestures only for back, home and overview -->
+    <integer name="config_navBarInteractionMode">0</integer>
```

Comment:
Does this just change the default mode or does it enable nav bar by default?

Reply:
It changes the default mode and thus disables the gestures, which are pretty annoying for me. The navbar itself doesnt show up (I switched manually while running your test build, didnt build myself)